app-office/taskcoach/taskcoach-1.4.6.ebuild

```diff
@@ -0,0 +1,45 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+PYTHON_COMPAT=( python2_7 )
+
+inherit desktop distutils-r1
+
+MY_PN="TaskCoach"
+MY_P="${MY_PN}-${PV}"
+
+DESCRIPTION="Simple personal tasks and todo lists manager"
+HOMEPAGE="https://www.taskcoach.org https://pypi.org/project/TaskCoach/"
+SRC_URI="mirror://sourceforge/${PN}/${MY_P}.tar.gz"
```

Comment:
I think using this mirror was deprecated, but I'm not sure.
@mgorny what do you say?

Reply:
There is #13105, but it seemed from the comments on that PR as if it was not yet settled that it would definitely happen.